software/controller/lib/debug/eeprom_cmd.cpp

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2020-2021, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include "commands.h"
+
+namespace Debug::Command {
+
+ErrorCode EepromHandler::Process(Context *context) {
+
+  // We have at least subcommand and address (3 bytes)
```

Comment:
Could we have an explicit definition of this subcommand+address data structure somewhere? Doesn't have to be a struct, but just to document it so we have a mental picture before reading all of the code.